src/api-server.ts

```diff
@@ -112,9 +113,10 @@ function subscribe(parameters: any, callback: any): void {
 */
 async function publish(parameters: any, callback: any): Promise<void> {
     //TODO if there's no active stream the server should warn the user
+    const sender = "111" //GET FROM metada
```

Comment:
I'd prefer not adding a sender if the 'sender' feature is not ready. I don't like the idea of a hardcoded sender "111"

Reply:
left a TODO, but and removed this.